staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go

```diff
@@ -453,7 +453,7 @@ func (e *Store) Update(ctx genericapirequest.Context, name string, objInfo rest.
 				return nil, nil, kubeerr.NewInvalid(qualifiedKind, name, fieldErrList)
 			}
 			if newVersion != version {
-				return nil, nil, kubeerr.NewConflict(e.QualifiedResource, name, fmt.Errorf(OptimisticLockErrorMsg))
+				return nil, nil, kubeerr.NewConflict(e.QualifiedResource, name, fmt.Errorf(optimisticLockErrorMsg, newVersion, version))
```

Comment:
We took this out originally because it was too busy - I'd prefer to add it as new structured fields in status under details rather than as a string